data/hai/hai reveal 4 middle.txt

```diff
@@ -1416,26 +1416,83 @@ mission "Hai Reveal [A14] A Time of Trouble"
+			choice
+				`	"Surely a group of enlightened human leaders can propose an alternative?"`
+				`	"Are there any counter-proposals?"`
+					goto "keep president happy"
+			`	The President seems genuinely irritated at being patronized, but the Chancellor smirks. You get a furry little smile from Teeneep.`
+			label "keep president happy"
+			`	"We have a counter-proposal," the President says. "We cannot afford to change our policy in totality, it would cost too many lives. However, we propose the following: The Free Worlds and Republic will establish a customs service on Mountaintop. Hai visiting human space will check in with us there. Our intelligence agencies will then track their movements in order to see to their safety as best we can. They'll also be given contacts on every human planet that they can go to for help. In the meantime, we will place extensive surveillance fleets at Ultima Thule to detect any unapproved Hai travel. We will do our best to disable and capture ships with kidnapped Hai, no matter the risk to Navy lives."`
 			`	"The Free Worlds will also extend our protection, of course," the Chancellor adds.`
-			`	Teeneep takes a moment to respond, "That sounds like an acceptable future state... but I cannot sign any treaty yet. The Hai will not be satisfied until our kidnapped brethren are returned."`
+			`	Teeneep looks relieved by this suggestion.`
+			choice
+				`	"That sounds like a viable solution that will benefit all parties. How does the Hai representative respond?"`
+				`	"Teeneep, will that satisfy the Hai?"`
+					goto "no human response"
+			`	Both human leaders nod in agreement with you.`
+			label "no human response"
+			`	Teeneep takes a moment to respond. "That sounds like an acceptable future state... but I cannot sign any treaty yet. The Hai will not be satisfied until our kidnapped brethren are returned."`
 			`	The President replies, "The Minister for Foreign Affairs has been keeping Cabinet apprised of the major details and proposals reported back by Deputy Minister Yang. Under the circumstances we believe it would be appropriate for us to attempt a joint effort to address your concerns. We agree to making this effort against one target, and to reassessing the situation afterwards depending upon what is discovered in that process. The Republic representatives there with you now will work with you to identify that target and plan the action accordingly."`
 			`	The smoothness with which this goes down gives you the sense that the main points of this were all provisionally agreed to already, with only a little wiggle room left to be settled in this conversation.`
 			`	Sayari steps up and whispers something to Teeneep who then seems to think for a moment before returning to the screens. "This pathway is acceptable."`
+			choice
+				`	(End the meeting.)`
```

Comment:
Another empty choice, as I explained in #7566.

Reply:
The player has three ways to end the conversation now.